docs/en/stack/ml/get-started/ml-gs-results.asciidoc

```diff
@@ -48,8 +48,8 @@ time. It is available only if the job has `model_plot_config` enabled. It can
 display only a single time series.
 
 The blue line in the chart represents the actual data values. The shaded blue
-area represents the bounds for the expected values. The area between the upper
-and lower bounds are the most likely values for the model. If a value is outside
+area represents the bounds for the expected values, which is also known as the _confidence interval_. The area between the upper
```

Comment:
Perhaps we should lose the `which is also known as the confidence interval` part of this sentence, as we don't use that label anywhere in the UI as far as I know, and then just leave '95% confidence level' in the next sentence.

Reply:
Agreed, I've made that change in c7d81b9678c8cf771661b913d05a4a3393e74e04